tools/data/hvu/parse_tag_list.py

```diff
@@ -0,0 +1,15 @@
+import mmcv
+
+tag_list = '../../../data/hvu/annotations/hvu_categories.csv'
+
+lines = open(tag_list).readlines()
```

Comment:
```
with open(tag_list, "r") as f:
    lines = f.readlines()
```

Reply:
Why performing this change? The change makes codes longer. btw, since we only read the tag_list (not write it), we do not need to close it. I think the advantage of using `with` is to automatically handle file closing when writing to a file.

Reply:
Agree.